Makefile.common

```diff
@@ -110,13 +112,17 @@ ARCH =  $(SIL)@mkdir -p $(@D); \
 	$(AR) $(ARFLAGS) $@ $^ 2>/dev/null
 
 LINK =  $(SIL)mkdir -p $(@D); \
-	(printf "LD  %-24s\n" "$(@F)"); \
+	(printf "LD  %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)"); \
```

Comment:
```suggestion
	printf "LD  %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)"; \
```
Brackets are unnecessary.

Reply:
not related to this PR, so leaving as-is. I agree that the parentheses are not needed in this case.